### PR TITLE
add dubbo3.x router workflow

### PR DESCRIPTION
**Pull request by chengyouling**
**What type of PR is this?**

dubbo3.x plugin test workflow.

**What this PR does / why we need it?**

In order to run dubbo3.x test action.

**Which issue(s) this PR fixes?**

https://github.com/huaweicloud/Sermant/issues/1452

**Does this PR introduce a user-facing change?**

No

### CODE REVIEW

**Comment by lilai23** [comment]
check DCO，see details on https://sermant.io/zh/document/community/contributing.html#%E5%BC%80%E6%BA%90%E8%B4%A1%E7%8C%AE%E5%8D%8F%E8%AE%AE

File: .github/actions/scenarios/dubbo/lane-dubbo3/action.yml [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+      if: matrix.registry-model == 'interface'
+      env:
+        dynamic.config.dynamicConfigType: KIE
+        dynamic.config.serverAddress: 127.0.0.1:30110
+        SERVICE_META_ENVIRONMENT: development
+        SERVICE_META_APPLICATION: lane
+        SERVICE_META_VERSION: 1.0.0
+        SERVER_PORT: 18021
+        DUBBO_PROTOCOL_PORT: 18821
+        DUBBO_REGISTRY_MODE: interface
```
Comment (lilai23) [inline .github/actions/scenarios/dubbo/lane-dubbo3/action.yml]: duplicate server_port

File: .github/actions/scenarios/dubbo/lane-dubbo3/action.yml [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+        dynamic.config.dynamicConfigType: KIE
+        dynamic.config.serverAddress: 127.0.0.1:30110
+        SERVICE_META_ENVIRONMENT: development
+        SERVICE_META_APPLICATION: lane
+        SERVICE_META_VERSION: 1.0.1
+        SERVER_PORT: 18022
+        DUBBO_PROTOCOL_PORT: 18822
+        DUBBO_REGISTRY_MODE: interface
+      run: |
+        nohup java -javaagent:sermant-agent-${{ env.sermantVersion }}/agent/sermant-agent.jar=appName=dubbo-integration-provider -Dsermant_log_dir=${{ env.logDir }}/provider2-${{ matrix.dubbo-version }}-${{ matrix.dubbo-versions }} -jar \
```
Comment (lilai23) [inline .github/actions/scenarios/dubbo/lane-dubbo3/action.yml]: check all the server_port. It should not be duplicate

File: .github/workflows/dubbo_integration_test.yml [hunk 3]
```diff
@@ -69,32 +69,129 @@ jobs:
     needs: [download-midwares-and-cache, build-agent-and-cache]
     strategy:
       matrix:
-        dubbo-version: ['2-5','2-6','2-7']
-        dubbo-versions: ['0','1','2','3','4','5','6','7','8','9','10','11','12']
+        registry-model: ['interface']
+        dubbo-version: ['2-5','2-6','2-7','3-0','3-1','3-2']
+        dubbo-versions: ['0','3','6','9','12']
```
Comment (lilai23) [inline .github/workflows/dubbo_integration_test.yml:74]: check your workflow result，3.0 3.1 3.2 run nothing for interface

File: .github/workflows/dubbo_integration_test.yml [hunk 4]
```diff
@@ -69,32 +69,129 @@ jobs:
...
             dubbo-versions: '3'
-          - dubbo-version: '2-7'
-            dubbo-versions: '4'
+            registry-model: 'interface'
+          - dubbo-version: '3-1'
+            dubbo-versions: '12'
+            registry-model: 'interface'
+          - dubbo-version: '3-2'
+            dubbo-versions: '12'
+            registry-model: 'interface'
```
Comment (lilai23) [inline .github/workflows/dubbo_integration_test.yml]: 3.0 3.1 3.2 run nothing for instance and all

File: .github/actions/scenarios/dubbo/lane-dubbo3/action.yml [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+      if: matrix.registry-model == 'all'
+      env:
+        dynamic.config.dynamicConfigType: KIE
+        dynamic.config.serverAddress: 127.0.0.1:30110
+        SERVICE_META_ENVIRONMENT: development
+        SERVICE_META_APPLICATION: lane
+        SERVICE_META_VERSION: 1.0.0
+        SERVER_PORT: 18021
+        DUBBO_PROTOCOL_PORT: 18821
+        DUBBO_REGISTRY_MODE: all
```
Comment (provenceee) [inline .github/actions/scenarios/dubbo/lane-dubbo3/action.yml]: try to use matrix.registry-model, and remove if

File: .github/actions/scenarios/dubbo/lane-dubbo3/action.yml [hunk 6]
```diff
@@ -0,0 +1,285 @@
+name: "Dubbo Lane Test"
```
Comment (provenceee) [inline .github/actions/scenarios/dubbo/lane-dubbo3/action.yml]: Dubbo3, check the name and description

File: sermant-plugins/sermant-router/dubbo3-router-plugin/src/test/java/com/huaweicloud/sermant/router/dubbo/interceptor/ServiceDiscoveryRegistryDirectoryInterceptorTest.java [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+    private final InstanceAddressURL instanceAddressURL;
+
+    /**
+     * 构造方法
+     */
+    public ServiceDiscoveryRegistryDirectoryInterceptorTest() {
+        RpcContext.getServiceContext().setConsumerUrl(APACHE_URL);
+        InvokerRuleStrategyService invokerRuleStrategyService = Mockito.mock(InvokerRuleStrategyService.class);
+        MockedStatic<PluginServiceManager> mockServiceManager = Mockito.mockStatic(PluginServiceManager.class);
```
Comment (provenceee) [inline sermant-plugins/sermant-router/dubbo3-router-plugin/src/test/java/com/huaweicloud/sermant/router/dubbo/interceptor/ServiceDiscoveryRegistryDirectoryInterceptorTest.java]: MockedStatic should be closed

File: .github/actions/scenarios/dubbo/lane-dubbo3/action.yml (hunk 1 above)
Comment (chengyouling) [reply .github/actions/scenarios/dubbo/lane-dubbo3/action.yml]: fixed

File: .github/actions/scenarios/dubbo/lane-dubbo3/action.yml (hunk 2 above)
Comment (chengyouling) [reply .github/actions/scenarios/dubbo/lane-dubbo3/action.yml]: fixed

File: .github/workflows/dubbo_integration_test.yml (hunk 3 above)
Comment (chengyouling) [reply .github/workflows/dubbo_integration_test.yml:74]: fixed

File: .github/workflows/dubbo_integration_test.yml (hunk 4 above)
Comment (chengyouling) [reply .github/workflows/dubbo_integration_test.yml]: fixed

File: .github/actions/scenarios/dubbo/lane-dubbo3/action.yml (hunk 5 above)
Comment (chengyouling) [reply .github/actions/scenarios/dubbo/lane-dubbo3/action.yml]: fixed

File: .github/actions/scenarios/dubbo/lane-dubbo3/action.yml (hunk 6 above)
Comment (chengyouling) [reply .github/actions/scenarios/dubbo/lane-dubbo3/action.yml]: fixed

File: sermant-plugins/sermant-router/dubbo3-router-plugin/src/test/java/com/huaweicloud/sermant/router/dubbo/interceptor/ServiceDiscoveryRegistryDirectoryInterceptorTest.java (hunk 7 above)
Comment (chengyouling) [reply sermant-plugins/sermant-router/dubbo3-router-plugin/src/test/java/com/huaweicloud/sermant/router/dubbo/interceptor/ServiceDiscoveryRegistryDirectoryInterceptorTest.java]: fixed

**Pull request closed without merging**